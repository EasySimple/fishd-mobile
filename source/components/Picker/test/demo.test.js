import webDemoTest from '../../../tests/demoTest';

webDemoTest('Picker');
