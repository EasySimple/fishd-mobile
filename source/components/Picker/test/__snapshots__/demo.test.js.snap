// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Picker demo correctly 1`] = `
<div>
  <div
    class="components-tpl-demo-basic"
  >
    <div
      class="demo-title"
    >
      基础用法
    </div>
    <div
      class="demo-card"
    >
      <div
        class="fm-list"
      >
        <div
          class="fm-list-body"
        >
          <span>
            <div
              class="fm-list-item fm-list-item-middle"
            >
              <div
                class="fm-list-line"
              >
                <div
                  class="fm-list-content"
                >
                  单列
                </div>
                <div
                  class="fm-list-extra"
                >
                  水果
                </div>
              </div>
              <div
                class="fm-list-ripple"
                style="display:none"
              />
            </div>
          </span>
          <span>
            <div
              class="fm-list-item fm-list-item-middle"
            >
              <div
                class="fm-list-line"
              >
                <div
                  class="fm-list-content"
                >
                  多列
                </div>
                <div
                  class="fm-list-extra"
                >
                  年份/季节
                </div>
              </div>
              <div
                class="fm-list-ripple"
                style="display:none"
              />
            </div>
          </span>
          <span>
            <div
              class="fm-list-item fm-list-item-middle"
            >
              <div
                class="fm-list-line"
              >
                <div
                  class="fm-list-content"
                >
                  级联
                </div>
                <div
                  class="fm-list-extra"
                >
                  地区选择
                </div>
              </div>
              <div
                class="fm-list-ripple"
                style="display:none"
              />
            </div>
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
